lua/jmod/sv_init.lua

```diff
@@ -89,7 +91,23 @@ hook.Add("Think","JMOD_SERVER_THINK",function()
 					playa.EZoxygen=math.Clamp(playa.EZoxygen+25,0,100) -- recover in 4 seconds
 				end
 			end
+			if(playa.EZblindness) then
+				local Blind = playa.EZblindness
+				if (Blind>0) then
+					playa.EZblindness = math.Clamp(playa.EZblindness-5,0,100)
+					if (math.random(1,100)<=playa.EZblindness/2.5) then
+						playa:EmitSound("vo/npc/male01/moan0"..math.random(1,5)..".wav",45,math.Rand(90,110))
+					end
+					if (math.random(1,100)<=playa.EZblindness/1.25) then
+						JMod_TryCough(playa)
+					end
+				end
+			end
 		end
+		net.Start("JMod_GasBlind")
+		if(playa.EZblindness) then net.WriteFloat(playa.EZblindness or 0) --[[ else net.WriteFloat(0) ]] end
+		net.Send(playa)
```

Comment:
gate this entirely on ezblindness, no need to waste bandwidth

Reply:
What does gate mean?